src/UploadedFile.php

```diff
@@ -114,7 +126,10 @@ public function __construct($streamOrFile, $size, $errorStatus, $clientFilename
     public function getStream()
     {
         if ($this->error !== UPLOAD_ERR_OK) {
-            throw new RuntimeException('Cannot retrieve stream due to upload error');
+            throw new RuntimeException(sprintf(
+                'Cannot retrieve stream due to upload error: %s',
+                self::$errorMessages[$this->error]
+            ));
```

Comment:
This can be tested by passing a non-`UPLOAD_ERR_OK` value to the `$errorStatus` constructor argument, and then calling this method.